web3/_utils/module_testing/personal_module.py

```diff
@@ -61,7 +66,87 @@ def test_personal_sendTransaction(self,
             'value': 1,
             'gasPrice': web3.toWei(1, 'gwei'),
         }
-        txn_hash = web3.personal.sendTransaction(txn_params, unlockable_account_pw)
+        txn_hash = web3.geth.personal.sendTransaction(txn_params, unlockable_account_pw)
+        assert txn_hash
+        transaction = web3.eth.getTransaction(txn_hash)
+
+        assert is_same_address(transaction['from'], txn_params['from'])
+        assert is_same_address(transaction['to'], txn_params['to'])
+        assert transaction['gas'] == txn_params['gas']
+        assert transaction['value'] == txn_params['value']
+        assert transaction['gasPrice'] == txn_params['gasPrice']
+
+    def test_personal_sign_and_ecrecover(self,
+                                         web3,
+                                         unlockable_account_dual_type,
+                                         unlockable_account_pw):
+        message = 'test-web3-geth-personal-sign'
+        signature = web3.geth.personal.sign(
+            message,
+            unlockable_account_dual_type,
+            unlockable_account_pw
+        )
+        signer = web3.geth.personal.ecRecover(message, signature)
+        assert is_same_address(signer, unlockable_account_dual_type)
+
+
+class ParityPersonalModuleTest():
+    def test_personal_listAccounts(self, web3):
+        accounts = web3.parity.personal.listAccounts()
+        assert is_list_like(accounts)
+        assert len(accounts) > 0
+        assert all((
+            is_checksum_address(item)
+            for item
+            in accounts
+        ))
+
+    def test_personal_unlockAccount_success(self,
+                                            web3,
+                                            unlockable_account_dual_type,
+                                            unlockable_account_pw):
+        result = web3.parity.personal.unlockAccount(
+            unlockable_account_dual_type,
+            unlockable_account_pw,
+            None
+        )
+        assert result is True
+
+    def test_personal_unlockAccount_failure(self,
+                                            web3,
+                                            unlockable_account_dual_type):
+        result = web3.parity.personal.unlockAccount(
+            unlockable_account_dual_type,
+            'bad-password',
+            None
+        )
```

Comment:
I cannot figure this out for the life of me why parity insists on returning `True` if there is a failure to the `personal_unlockAccount` call.

Reply:
You could a comment to the code explaining that this is the actual expected output at the moment, since its pretty weird.  Maybe open an issue with Parity?